src/components/virtual-assessment-request/VirtualAssessmentRequest.tsx

```diff
@@ -185,12 +190,17 @@ const VirtualAssessmentRequestModal = ({
                     variant="outlined"
                     color="primary"
                     fullWidth
+                    disabled={new Date(appointment.startDate) < CURRENT_DAY}
```

Comment:
are you sure that it will work correctly with timezones, because curent day stores current pc time

Reply:
fixed